management-api/base/deploymentconfig.yaml

```diff
@@ -31,6 +31,11 @@ spec:
               value: "0"
             - name: THOTH_MANAGEMENT_API_RUN_MIGRATIONS
               value: "0"
+            - name: THOTH_API_VALIDATE_RESPONSES
+              valueFrom:
+                configMapKeyRef:
+                  key: management-api
+                  name: thoth
```

Comment:
will this irritate cnv-prod? as the configmap does not have the required key?! @harshad16

Reply:
I put `validate-responses` to the base configmap which is not patched for prod deployment. But maybe I did something wrong?

Reply:
I think , this is good enough. As it an environment variable , so even if it is present and the management API version is not consuming it, it would not break the deployment. 
however the envvar referencing seems to be misconfigured in here.